<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2" rowspan="4" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Total crRNA View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QGraphicsView" name="total_crRNA"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Individual crRNA View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="single_crRNA"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QPushButton" name="clear_selection">
      <property name="text">
       <string>Clear Selection</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTableWidget" name="crRNA_table">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>10000000</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Step 1: Input  Data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2" colspan="4">
        <widget class="QGroupBox" name="groupBox_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Selected File:</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="path_1">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Off-Target Query File:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Selected File:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="path_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="browse_3">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Selected File:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="path_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="browse_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3" rowspan="2" colspan="3">
        <widget class="QProgressBar" name="progressBar">
         <property name="minimumSize">
          <size>
           <width>430</width>
           <height>0</height>
          </size>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="browse_1">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Off-Target Reference File:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>crRNA Sequence File:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="2" colspan="3">
        <widget class="QPushButton" name="run_OT">
         <property name="minimumSize">
          <size>
           <width>215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Run Off-Target</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="plot_selection">
      <property name="text">
       <string>Plot  Selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export:</string>
    </property>
    <addaction name="actionSelected_crRNAs_to_CSV"/>
    <addaction name="actionAll_crRNAs_to_CSV"/>
   </widget>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelected_crRNAs_to_CSV">
   <property name="text">
    <string>Selected crRNAs to CSV</string>
   </property>
  </action>
  <action name="actionAll_crRNAs_to_CSV">
   <property name="text">
    <string>All crRNAs to CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
